package.json

```
@@ -165,7 +167,7 @@
     "spell-check": "file:packages/spell-check",
     "status-bar": "file:packages/status-bar",
     "styleguide": "file:./packages/styleguide",
-    "superstring": "^2.4.4",
+    "superstring": "github:pulsar-edit/superstring",
```

Likewise, want to specify the exact SHA here.

```suggestion
    "superstring": "github:pulsar-edit/superstring#de97b496663fce40050bf2d66e1466ccfbd00943",
```